nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -49,6 +50,14 @@
 
     HAVE_APEX = False
 
+try:
+    import logging
+    from lddl.torch_mp import get_bert_pretrain_data_loader
```

Comment:
Can  we add the library to requirements file in Nemo. @ericharper

Reply:
@wdykas

Reply:
yes, add it to requirements_nlp.txt

Reply:
done